topology/probes/netlink.go

```diff
@@ -689,6 +690,10 @@ Ready:
 						}
 						tr.Metadata["Statistics"] = u.statsToMap(stats)
 						lastMetrics := u.statsToMap(&metric)
+						if reflect.DeepEqual(tr.Metadata["Statistics"], lastMetrics) {
```

Comment:
I think you are comparing the wrong values (absolute vs relative)

You could do this instead :

changed := true
for i, v := range lastMetrics {
    if v != 0 {
        break
    } else if i == len(lastMetrics) - 1 {
        changed = false
    }
}
if !changed {
    u.Graph.Unlock()
    continue
}